sami2py/fortran/exb.inp

```diff
@@ -0,0 +1,10 @@
+0.000000000000000000e+00 0.000000000000000000e+00
```

Comment:
So i had to change the git ignore in my other pull request, because it was looking for "ExB.inp". However the file created was all lowercase. So in the other pull, I changed the gitignore to reflect that.